tempesta_fw/sock.c

```diff
@@ -210,13 +210,13 @@ ss_send(struct sock *sk, SsSkbList *skb_list, bool pass_skb)
 		ss_skb_queue_head_init(&sw.skb_list);
 		for (skb = ss_skb_peek(skb_list); skb; skb = ss_skb_next(skb)) {
 			/* tcp_transmit_skb() will clone the skb. */
-			skb_copy = pskb_copy_for_clone(skb, GFP_ATOMIC);
-			if (!skb_copy) {
+			twin_skb = pskb_copy_for_clone(skb, GFP_ATOMIC);
```

Comment:
I renamed it as I experimented with different SKB copy functions, one of which is `skb_copy()`. That caused the name conflict. I suggest that we keep this change.